### PR TITLE
chore(workflow-rc1): releasing workflow-rc1(-e2e)

### PR DESCRIPTION
**Pull request by smothiki**
deis charts for RC1

### CODE REVIEW

File: workflow-rc1-e2e/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: workflow-rc1-e2e
+home: https://github.com/deis/workflow-e2e
+version: v2-beta
```
Comment (mboersma) [inline workflow-rc1-e2e/Chart.yaml]: version should be "v2-rc1" I assume?

File: workflow-rc1/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+name: workflow-rc1
+home: https://github.com/deis/workflow
+version: v2-beta
```
Comment (mboersma) [inline workflow-rc1/Chart.yaml]: `v2-rc1`?

File: workflow-rc1/manifests/deis-namespace.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: deis
+  labels:
+    heritage: deis
+  annotations:
+    helm-keep: "true"
```
Comment (mboersma) [inline workflow-rc1/manifests/deis-namespace.yaml:8]: 👍

File: workflow-rc1-e2e/Chart.yaml (hunk 1 above)
Comment (jchauncey) [inline workflow-rc1-e2e/Chart.yaml]: yeah that should be v2.0.0-rc1

Comment (vdice) [inline workflow-rc1-e2e/Chart.yaml]: will file a ticket in `deisrel` to catch this and the one below..

Comment (vdice) [inline workflow-rc1-e2e/Chart.yaml]: fyi https://github.com/deis/deisrel/issues/66

File: workflow-rc1/Chart.yaml (hunk 2 above)
Comment (vdice) [inline workflow-rc1/Chart.yaml]: yup, good catch; issue filed in deisrel to catch this next time: https://github.com/deis/deisrel/issues/66

File: workflow-rc1-e2e/tpl/workflow-e2e-pod.yaml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        value: "{{env "GINKGO_NODES" | default 15}}"
+      - name: JUNIT
+        value: "true"
+    volumeMounts:
+    - name: artifact-volume
+      mountPath: /root
+  - name: artifacts
+    image: busybox
+    imagePullPolicy: Always
+    command: ["tail", "-f", "/rc1/null"]
```
Comment (jchauncey) [inline workflow-rc1-e2e/tpl/workflow-e2e-pod.yaml]: this needs to be changed to be `/dev/null`

Comment (smothiki) [inline workflow-rc1-e2e/tpl/workflow-e2e-pod.yaml]: rofl. Updating

**Pull request merged**